src/Riok.Mapperly.Abstractions/MapperAttribute.cs

```diff
@@ -38,9 +38,19 @@ public sealed class MapperAttribute : Attribute
     /// Specifies the behaviour in the case when the mapper tries to set a non-nullable property to a <c>null</c> value.
     /// If set to <c>true</c> an <see cref="ArgumentNullException"/> is thrown.
     /// If set to <c>false</c> the property assignment is ignored.
+    /// This is ignored for required init properties and <see cref="IQueryable{T}"/> projection mappings.
     /// </summary>
     public bool ThrowOnPropertyMappingNullMismatch { get; set; }
 
+    /// <summary>
+    /// Specifies whether <c>null</c> values are assigned to the target.
+    /// If <c>true</c>, the source is <c>null</c>, and the target does allow <c>null</c> values,
+    /// <c>null</c> is assigned.
+    /// If <c>false</c>, <c>null</c> values are never assigned to the target property.
+    /// This is ignored for required init properties and <see cref="IQueryable{T}"/> projection mappings.
+    /// </summary>
+    public bool AllowNullPropertyAssignment { get; set; } = true;
```

Comment:
I think we should explicitely document that this has a default value of `true` in the XML doc. Makes it easier for users, as they do not have to consult the documentation/code to see what the default value is.

Reply:
Fixed.